### PR TITLE
adding more ML mock up code

### PR DESCRIPTION
**Pull request by agomoll**
Adding more mock up code for the ML models. Notice the commented out block of Deep learning code. Tensorflow is killing my new 'loaner' computer with the new mac M1 chip. I haven't installed the required packages yet...

### CODE REVIEW

**Comment by bdod92** [comment]
1. In your saved outputs, the random forest model gave ~90% balanced accuracy. When I ran it without any changes, it had a 79% so there is significant variability.
2. After the SMOTE adjustment, the logistic regression has an error when running due to the model having the wrong name. y_smote_pred = model.predict should be changed to y_smote_pred = smote_model.predict
3. RandomForestClassifier also needs to be imported (sklearn) to run the random forest section (after balancing with SMOTE). Currently, only balanced random forest is imported.
